validator/packaging/validator.toml.example

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
Maybe we should have a unit test which makes sure the values here reflect the actual default values to prevent them from drifting auto-of-sync.

The defaults are probably scattered around various modules currently; we should probably move them to config/validator.py (load_default_validator_config?) and merge them in as well.